.coveragerc

```diff
@@ -23,7 +23,9 @@ exclude_lines =
     if 0:
     if False:
     if __name__ == .__main__.:
-    
+    if typing.TYPE_CHECKING:
+    if TYPE_CHECKING:
+
```

Comment:
What is happening with the whitespace here? Shouldn't this change appear as a simple +2/-0 diff?

Reply:
I've no idea.

Reply:
I figured it out. There was whitespace on the empty line before, which GitHub isn't showing even though I have "Ignore whitespace" turned off in diffs. This being +3/-1 is a good thing.